README.md

```diff
@@ -213,7 +213,45 @@ The current version of Lumberjack requires:
 - If you **have a feature request**, open an issue.
 - If you **want to contribute**, submit a pull request.
 
+### Data Collection Practices
+
+Per  [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/), Apple is requesting app developers to provide info about their data collection, us SDK maintainers must provide them with the same data.
+
+#### Data collection by the framework
+
+**By default, CocoaLumberjack does not collect any data on its own.** 
+[See our updated list using the recommended format.](https://cocoalumberjack.github.io/DataCollection/index.html)
+
+#### Indirect data collection through the framework
+
+Considering CocoaLumberjack is a logging framework which makes it easy to send those logs to different platforms, collecting  data might happen quite easily, if the app developers include any sensitive data into those logs.
+
+Private data includes but isn't limited to:
+
+- user info (name, email, address, ...)
+- location info
+- contacts
+- identifiers (user id, device id, ...)
+- app usage data
+- performance data
+- health and fitness info
+- financial info
+- sensitive info
+- user content
+- history (browsing, search, ...)
+- purchases
+- diagnostics
+- ...
+
+Example: `DDLogInfo("User: \(myUser)")` will add the `myUser` info to the logs, so if those are forwarded to a 3rd party or sent via email, that may qualify as data collection.
+
+**The important thing is you app developers are fully responsible for adding sensitive data to the logs, thus documenting that via the Data Collection Practices info!**
```

Comment:
Here, I would like to add section Important?

### Important note

```
App developers are fully responsible...
```

Reply:
Good idea, I have changed that am moved it up

Reply:
I still not see it. Look at `rich diff view`.

If I read it in text, I won't spend enough attention/time on it. But I should.
**Important note: app developers are fully responsible for any sensitive data collected through our logging system!**